lib/graphql/anycable/config.rb

```diff
@@ -5,13 +5,49 @@
 module GraphQL
   module AnyCable
     class Config < Anyway::Config
+      ALLOWED_DELIVERY_METHODS = %w[inline active_job].freeze
+
       config_name :graphql_anycable
       env_prefix  :graphql_anycable
 
       attr_config subscription_expiration_seconds: nil
       attr_config use_redis_object_on_cleanup: true
       attr_config use_client_provided_uniq_id: true
       attr_config redis_prefix: "graphql" # Here, we set clear redis_prefix without any hyphen. The hyphen is added at the end of this value on our side.
+
+      attr_config delivery_method: "inline", queue: "default", job_class: "GraphQL::Jobs::TriggerJob"
+
+      def job_class=(value)
```

Comment:
Maybe here, can we add a checker if the "job_class" is not corrected? Check not only blank

For example, the user puts job_class as "SpmeNonExistentClass", and we load it through `safe_constantize`

If it returns `nil`, we will raise an error, something like "job_class has non-existence class"